### ISSUE TITLE
I'm listed 2x on the all_time list 

### ISSUE THREAD

**Issue by carols10cents** [rust-lang-nursery/thanks#102]
Related: https://github.com/rust-lang-nursery/thanks/issues/45
I should be fixed in the mailmap: https://github.com/steveklabnik/rust/commit/0f2a5f686f3fbc3d66dce301e965b43d5c3acd63

But I'm listed currently at number 100 and number 1231 :-/ https://thanks.rust-lang.org/rust/all-time

There may be more: https://twitter.com/llogiq/status/896319084685611008

> Are there really two Brian Andersons? # 2 and # 85

**Comment by steveklabnik** [comment]
Maybe I need to regenerate the DB...

**Comment by russmack** [comment]
@carols10cents _"I'm pretty sure my git emails match."_ -- one is math related.

**Comment by carols10cents** [comment]
@russmack yes, the mailmap should take care of that https://github.com/steveklabnik/rust/commit/0f2a5f686f3fbc3d66dce301e965b43d5c3acd63

**Comment by russmack** [comment]
Hi @carols10cents , you'll be unified at the next merge: https://github.com/rust-lang/rust/pull/47637 as will Brian.
